index.html

```diff
@@ -76,35 +76,39 @@ <h2 class="underline-headline">Paralax section</h2>
           </p>
           <p class="quote-author">— Erik Adigard</p>
         </div>
+        <div class="bottom-picture"></div>
       </div>
       <div class="commercial-data">
         <ul class="list-of-data">
-          <li>
+          <li class="list-of-data-li">
             <div class="chapter">
-              <img src="src/img/user.png" alt=""/>
-              <p class="list-numbers">1000+</p>
+              <div class="chapter-container"><img src="src/img/user.png" alt="" class="chapter-image"/>
+              <p class="list-numbers">1000+</p></div>
               <p class="list-letters">Happy users</p>
             </div>
           </li>
-          <li>
-            <div class="chapter">
-              <img src="src/img/cart.png" alt="" />
-              <p class="list-numbers">200</p>
-              <p class="list-letters">Sold products</p>
+          <li class="list-of-data-li">
+            <div class="chapter-two">
```

Comment:
Use common class `chapter` and modificators

Reply:
Done